Tools/scripts/configure-ci.sh

```diff
@@ -83,5 +83,5 @@ fi
 
 . ~/.profile
 
-pip install --user -U argparse empy pyserial pexpect future lxml
+pip install --user -U argparse empy pyserial pexpect future lxml numpy
```

Comment:
Ah, so it was you that changed our Semaphore configuration! I changed it back, sorry - we don't need two CIs running sitltest.

This is fine, but I doubt that's all you need - it had failed anyway (https://semaphoreci.com/diydrones/ardupilot/branches/pull-request-5651/builds/1). Also, realize that with Semaphore it's easy to have different commands for your repo.